pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -678,13 +678,17 @@ type PGMonitorSpec struct {
 type ExporterSpec struct {
 
 	// Projected volumes containing custom PostgreSQL Exporter configuration.  Currently supports
-	// the customization of PostgreSQL Exporter queries. If a "queries.yaml" file is detected in
+	// the customization of PostgreSQL Exporter queries. If a "queries.yml" file is detected in
 	// any volume projected using this field, it will be loaded using the "extend.query-path" flag:
 	// https://github.com/prometheus-community/postgres_exporter#flags
 	// Changing the values of field causes PostgreSQL and the exporter to restart.
 	// +optional
 	Configuration []corev1.VolumeProjection `json:"configuration,omitempty"`
 
+	// Projected secret containing custom TLS certificates to encrypt output from the exporter
+	// web server
+	CustomTLSSecret *corev1.SecretProjection `json:"customTLSSecret,omitempty"`
```

Comment:
🔧 `omitEmpty` implies `+optional`, but we should include the marker so people don't have to know that.